src/components/Cards/ResourceCard.tsx

```diff
@@ -20,21 +20,24 @@ const ResourceCard = (props: ResourceCardProps): typeof ResourceCard => {
 
   if (type == 'Article') {
     return (
-      <Card className={classes.articleCard}>
-        <h3 className={classes.title}>{resource.title}</h3>
-        <text className={classes.text}>{resource.data.preview}</text>
-      </Card>
+      <InstructionCard
+        resource={props.resource}
+        resourceID={props.resourceID}
+        expand={props.expand}
```

Comment:
Do the `expand` and `includeCheck` props on `InstructionCard` have the same purpose as `viewAll` and `completeCheck` on `VideoCard`? If so, I think we should name them both the same way for consistency!

Reply:
yes!! fixed <3